aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -263,6 +287,16 @@ module AptosFramework::Coin {
         }
     }
 
+    public fun initialize_coin_events(account: &signer) {
+        assert!(
+            !exists<CoinEvents>(Signer::address_of(account)),
+            Errors::not_published(ECOIN_EVENTS_ALREADY_PUBLISHED),
+        );
```

Comment:
then you can delete this, since it was already established it didn't exist.

Reply:
Deleted I had it originally as this fun was public (I was thinking someone can do this independently of register/initialize) but it wasn't really that useful of a use case. Removed the check.